index.html

```diff
@@ -23,5 +25,188 @@ <h2>I’am a Software Developer.</h2>
         <li><a href="https://medium.com/@nketchoguemilton"><img src="images/medium.svg" alt="Medium" width="55" height="55" /></a></li>
       </ul>
     </section>
+
+  <section id="projects">
+    <div>
+      <h1>My Recent Works</h1>
+    </div>
+    <div>
+      <div id="flex-projects">
+        <div>
+          <img src="./images/gymfit.png" alt="gymfit" />
+        </div>
+        <div>
+          <h2>Multi-Post Stories</h2>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard dummy text
+            ever since the 1500s, when an unknown printer took a standard
+            dummy text. A daily selection of privately personalized reads; no
+            accounts or sign-ups required.
+          </p>
+          <div class="lang-box">
+            <span>css</span>
+            <span>html</span>
+            <span>bootstrap</span>
+            <span>ruby</span>
+          </div>
+          <a href=""></a><button>See Project</button></a>
+        </div>
+        <div class="demo-wrap">
+          <div class="main-projects  demo-content">
+            <h3>Professional Art<br />Printing Data</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
```

Comment:
- [ ]  Kindly follow the styling  requirements for the project
<img width="504" alt="Screen Shot 2022-10-29 at 05 24 07" src="https://user-images.githubusercontent.com/81444338/198831515-864ba109-aa3e-4813-b737-de4d727acbca.png">